server.js

```diff
@@ -1,25 +1,19 @@
-app.put('/slambook/:id', async (req, res) => {
+app.delete('/slambook/:id', async (req, res) => {
   const entryId = req.params.id;
 
   try {
-   

```

Comment:
The closing brace of the callback function should be on a new line for better readability.